### ISSUE TITLE
Adding "short_name" field to manifest.json

### ISSUE THREAD

**Issue by e-com-presta** [thirtybees/thirtybees#1307]
https://github.com/thirtybees/thirtybees/blob/cc2e9aea5c23c95e18610c69392534429037bcec/controllers/admin/AdminThemesController.php#L2195
'short_name'       => Configuration::get('PS_SHOP_NAME'),

**Comment by e-com-presta** [comment]
Lighthouse Audit reports an error if there is no 'short_name' field in the manifest.

**Comment by getdatakick** [comment]
Yeah, they do. They have more strict validation rule than needed. 

According to specification, this field is NOT mandatory: 

- https://developer.mozilla.org/en-US/docs/Web/Manifest/short_name
- https://developer.chrome.com/docs/extensions/mv2/manifest/name/#short_name

It's nice to have, but not required. Therefore this is enhancement, not the bug. 

Also, just using `Configuration::get('PS_SHOP_NAME')` is probably not good enough solution, as the `short_name` recommended max length is 12 chars. Even though this is just a recommendation and not a requirement, lighthouse would fail validation on this as well.
